### DISCUSSION TITLE
我的图片这一栏无权限访问 这是怎么回事？

### DISCUSSION THREAD

**Discussion by heiyi110** [lsky-org/lsky-pro#598] [问答]
我的图片这一栏无权限访问    提示：
Forbidden
You don't have permission to access this resource.

**Comment by mumuxi-git** [comment]
求个答案，我的也是这样，不知道该调整哪里

**Comment by mumuxi-git** [comment]
试了一下，储存策略中的访问网址之前为`域名/images`，修改为`域名/i`就可以正常打开了，改为其他比如`pic` `img`都没问题，就是`/images`不行，估计是和哪里有冲突吧。
